apiserver/encrypting-data-at-datastore-layer.md

```diff
@@ -0,0 +1,285 @@
+---
+title: Encrypting Data at Datastore Layer
+authors:
+  - "@enj"
+reviewers:
+  - "@sttts"
+  - "@deads2k"
+  - "@p0lyn0mial"
+approvers:
+  - "@sttts"
+  - "@derekwaynecarr"
+  - "@smarterclayton"
+creation-date: 2019-09-09
+last-updated: 2019-09-20
+status: implementable
+see-also:
+  - "/enhancements/etcd/storage-migration-for-etcd-encryption.md"
+---
+
+# Encrypting Data at Datastore Layer
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift/docs]
+
+## Summary
+
+Provide automatic and seamless support for encryption of data stored in etcd while maintaining the principles of OS4 (a self-managed platform that abstracts cluster configuration through intuitive Kubernetes style APIs).
+
+## Motivation
+
+Even in the presence of full disk encryption, users want the ability to encrypt data stored in etcd.  This provides an extra layer of protection against data leakage.  For example, it protects the loss of sensitive secret information if an etcd backup is exposed to the incorrect parties.
```

Comment:
non-native-speaker-nit: protect against the loss

Reply:
> non-native-speaker-nit: protect against the loss

@enj take his suggestion.